### ISSUE TITLE
SproutSocial and Social Media Champions

### ISSUE THREAD

**Issue by amiller-gh** [nodejs/community-committee#411]
@ZibbyKeaton to write proposal and present next meeting 🎉

**Comment by bnb** [comment]
Super excited about this. Since I know a lot of y'all probably don't have experience with SproutSocial, I'd like to share a bit of my experience with it:

I personally use it to manage the NodeSource social properties. Of all the tools that we could _possibly_ use to this, I am happy to hear that the LF uses it for Node.js social properties and that we can possibly integrate Social Media Champions into that workflow.

Before using it at NodeSource, we used a built-in tool in HubSpot. When we were moving away from HubSpot, I was tasked with evaluating social tools. Buffer, HootSuite, MeetEdgar, and a handful of others... SproutSocial won, hands down. I've used it for the last year or so, and would not want to use another one. Really good UX for an "enterprise" tool when compared to the other options.

I know y'all probably weren't looking for a testimonial, but wanted to share my experience and trust in this tool for the goal we're trying to achieve 🤗
